src/main/java/bio/terra/service/dataset/flight/LockDatasetStep.java

```diff
@@ -17,10 +18,16 @@
 
     private final DatasetDao datasetDao;
     private final UUID datasetId;
+    private boolean suppressNotFoundException; // default to false
```

Comment:
Can you add a comment on when you would want to suppress a `DatasetNotFoundException` vs when you would want to error out instead?

Reply:
Done. Added comment to constructor for LockSnapshotStep and LockDatasetStep.